### PR TITLE
WIP: Require Julia 0.7-alpha to run Femtocleaner

### PR DESCRIPTION
**Pull request by nalimilan**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-julia 0.7.0-alpha
+julia 0.7-alpha
```
Comment (ararslan) [inline REQUIRE]: Why is this necessary? Seems like 0.7.0-alpha should be sufficient, as it's the same as 0.7-alpha.

Comment (nalimilan) [reply REQUIRE]: Looks like neither works... :-/

Comment (ararslan) [reply REQUIRE]: ...welp.

Comment (nalimilan) [reply REQUIRE]: CC: @Keno

Comment (Keno) [reply REQUIRE]: 0.7.0-alpha would have been fine. Looks like femtocleaner got itself stuck somewhere. I'll restart it for now.

Comment (Keno) [reply REQUIRE]: Try again (you can close and re-open the PR to re-trigger the check).

Comment (Keno) [reply REQUIRE]: Hmm, it's possible that this package itself causes it to hang. I'll take a look.

Comment (Keno) [reply REQUIRE]: Nope, there it is. It just takes some time.

Comment (nalimilan) [reply REQUIRE]: Thanks, it worked! It even finished in -1m :-)

Comment (Keno) [reply REQUIRE]: It tried to do the following:
```
  ## histograms ##
  function sturges(n)  # Sturges' formula
      n==0 && return one(n)
-     ceil(Integer, digits=log2(n))+1
+     ceil(Integer, digits=digits=log2(n))+1
  end
I think you can see where that'd go. I'll deploy a fix and report back.
```

Comment (Keno) [reply REQUIRE]: I have deployed https://github.com/JuliaComputing/Deprecations.jl/commit/42ca003101a57341046acd8f2bc3528d1e064cc3 which fixes that particular issue. If you hit the "Fix it" button again, you should be able to trigger femtocleaner to get you a fixed commit.

**Comment by ararslan** [comment]
I have things working in #394 (in retrospect I should have picked it up from here), so I'll close this in favor of that PR.
